chain/chain/src/metrics.rs

```diff
@@ -15,6 +15,13 @@ pub static BLOCK_PROCESSED_SUCCESSFULLY_TOTAL: Lazy<IntCounter> = Lazy::new(|| {
     )
     .unwrap()
 });
+pub static BLOCK_SUCCESSFUL_PROCESSING_TIME: Lazy<Histogram> = Lazy::new(|| {
+    try_create_histogram(
+        "near_block_successful_processing_time",
+        "Time taken to successfully process blocks",
+    )
+    .unwrap()
+});
 pub static BLOCK_PROCESSING_TIME: Lazy<Histogram> = Lazy::new(|| {
     try_create_histogram("near_block_processing_time", "Time taken to process blocks").unwrap()
 });
```

Comment:
```
BLOCK_PROCESSED_TOTAL
BLOCK_PROCESSED_SUCCESSFULLY_TOTAL
BLOCK_SUCCESSFUL_PROCESSING_TIME
BLOCK_PROCESSING_TIME
```

successs/no-succes order of statcis is inconsistent

word-order within is also inconsistent, but not sure how to fix that